<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>579</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>579</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>작성하기</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>561</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>681</width>
     <height>561</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>681</width>
     <height>561</height>
    </size>
   </property>
   <widget class="QPlainTextEdit" name="Title">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Descrip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>661</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>239</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="BackButton">
       <property name="text">
        <string>이전</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="NextButton">
       <property name="text">
        <string>다음</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="CheckButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>정답확인</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Date">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
